mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,40 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
```

Comment:
That is really cool.

did a little alteration to it, see if you like my decoration